docs/aws-professional/services.md

```diff
@@ -7,13 +7,13 @@ ms.date: 03/24/2017
 
 pnp.series.title: Azure for AWS Professionals
 ---
-# AWS to Azure services comparison
+# Azure & AWS Cloud Service Map
```

Comment:
imo the original title is more clear

Reply:
I see this as more of a "mapping" between service vs. a comparison. If it was a comparison, I'd expect AWS does x and Azure does y.